README.md

```diff
@@ -1,61 +1,57 @@
 # ansible-aci
 
-The ansible-aci project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
+## Description
 
-This collection has been tested and supports ACI 3.2+.
-Modules supporting new features introduced in ACI API in specific ACI versions might not be supported in earlier ACI releases.
+The `ansible-aci` project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
 
-*Note: This collection is not compatible with versions of Ansible before v2.8.*
```

Comment:
I think, we should still keep this note somewhere in readMe

Reply:
We already mention in the requirements section that the collection requires Ansible v2.15+
v2.8 has been EOL for almost 3 years now. I think its okay to remove.